charts/ceph-csi-cephfs/README.md

```diff
@@ -2,7 +2,25 @@
 
 The ceph-csi-cephfs chart adds cephfs volume support to your cluster.
 
-## Install Chart
+## Install from release repo
+
+Add chart repository to install helm charts from it
+
+```console
+helm repo add ceph-csi https://ceph.github.io/csi-charts
+```
+
+## Install from local Chart
+
+we need to enter into the directory where all charts are present
+
+```console
```

Comment:
This assumes the ceph-csi git repository is already cloned and the current working directory. I am not sure that is reasonable. Maybe users need the commands to clone the repository too?

Reply:
This assumes that the user has cloned the repo. do you want me to add code to clone and check out the repo? I am assuming that user will  know these steps

Reply:
Ok, if that is your assumption, no need to add it now. If someone wants to add them in the future, that can always be done.